migrations/2020/Version20200214114359.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20200214114359 extends AbstractMigration
+{
+    public function getDescription() : string
+    {
+        return 'Unify mapping between Doctrine and ac_* tables';
+    }
+
+    public function up(Schema $schema) : void
+    {
+        $this->addSql('DROP INDEX participantId_event_type ON ac_participants');
+        $this->addSql(<<<SQL
+            ALTER TABLE ac_participants
+                CHANGE id id CHAR(36) NOT NULL COMMENT '(DC2Type:payment_id)',
+                CHANGE participantId participantId INT UNSIGNED NOT NULL,
+                CHANGE event_id event_id INT NOT NULL,
+                CHANGE payment payment NUMERIC(8, 2) NOT NULL COMMENT '(DC2Type:money)',
```

Comment:
money ma byt typu int ne?

Reply:
Ty migrace jdou postupně, aby to bylo přehlednější. flost -> int se dělá až v té další

Reply:
ok